packages/tooltip/src/Tooltip.spec.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import 'jest-dom/extend-expect';
+import { render, fireEvent, cleanup } from '@testing-library/react';
+import Tooltip from '.';
+
+afterAll(cleanup);
+
+describe('packages/Tooltip', () => {
+  const buttonText = 'trigger button';
+
+  describe('when uncontrolled', () => {
+    const triggerEvent = 'click';
+
+    const { getByText, getByTestId } = render(
+      <>
+        <div>backdrop content</div>
+        <Tooltip
+          trigger={<button>{buttonText}</button>}
+          triggerEvent={triggerEvent}
+        >
+          <div data-testid="uncontrolled-tooltip">Uncontrolled tooltip!</div>
+        </Tooltip>
+      </>,
+    );
+
+    const button = getByText(buttonText);
+    const tooltip = getByTestId('uncontrolled-tooltip');
+    const backdrop = getByText('backdrop content');
+
+    test(`renders a button to the DOM with ${buttonText}`, () => {
+      expect(button).toBeInTheDocument();
+      expect(button.tagName.toLowerCase()).toBe('button');
+    });
+
+    test(`${triggerEvent} triggers opening and closing of tooltip`, () => {
```

Comment:
We need a test that clicking on the content inside of the tooltip content does not cause the tooltip to hide.

Reply:
👍